.testcoverage.yml

```diff
@@ -24,7 +24,7 @@ threshold:
 
```

Comment:
So far I tested various actions:
- applied several dynakubes with many combinations (deployment modes etc)
- tried basic stuff like querying dynakubes, looking at the actual manifests
- unit tests
- currently running e2e tests

All looks good so far to me. Codewise I do not have anything to add further. From my side there would be an approval, if CI is fixed. But let's wait maybe anyone also wants to review and finds something.